gnome-extra/gnome-builder/gnome-builder-3.16.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+GCONF_DEBUG="no"
+
+inherit gnome2
+
+DESCRIPTION="Builder attempts to be an IDE for writing software for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/Builder"
+
+LICENSE="GPL-3+ GPL-2+ LGPL-3+ LGPL-2+ MIT CC-BY-SA CC0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="introspection"
+
+RDEPEND="
+	>=dev-libs/gjs-1.42
+	>=dev-libs/glib-2.44
```

Comment:
Please add slots on all dependencies that have one. It is important to avoid extra work when a new slot is introduced.